bin/start.js

```diff
@@ -1,10 +1,19 @@
 require('@ministryofjustice/module-alias/register-module')(module)
+
 const server = require('~/fb-runner-node/server/server')
 
 const debug = require('debug')
 const log = debug('runner:start:log')
 const error = debug('runner:start:error')
 
+const {
+  env: {
+    DEBUG = 'runner*,client*'
+  }
+} = process
+
+debug.enable(DEBUG)

```

Comment:
Pick up the `DEBUG` env and programmatically set that (with a default) when the application is started

Reply:
(This ensures that `debug` will log to `stdout` in acceptance tests)